File: mysql-nodejs-music/previous/musicretrieveall.js
var mysql = require('mysql');

var con = mysql.createConnection({
  host: "localhost",
  user: "root",
  database: "music"
});

con.connect(function(err) {
  if (err) throw err;
  con.query("SELECT * FROM musicians", function (err, result, fields) {
    if (err) throw err;
    console.log(result);
  });
});

File: mysql-python-music/musicretrieveall.py
import mysql.connector

# Execute:
# python3 musicretrieveall.py
#

mydb = mysql.connector.connect(
  host="localhost",
  user="root",
  passwd="",
  database="music"
)

mycursor = mydb.cursor()

mycursor.execute("SELECT * FROM musicians")

myresult = mycursor.fetchall()

for x in myresult:
  print(x)
